### ISSUE TITLE
3DSSD training loss

### ISSUE THREAD

**Issue by turboxin** [tomztyang/3DSSD#24]
Hi, thanks for sharing this fantastic work!

I'm trying to implement 3DSSD in pytorch, but the loss of vote layer didn't seem to converge (4e4~5e4), while the loss of head is 0.5. I'm trying to figure out what I have done wrong, could anyone provide a log file or a tensorboard file of the 3DSSD training process. Thanks a lot!
